addons/medical/functions/fnc_treatment.sqf

```diff
@@ -226,6 +226,69 @@ private _treatmentTime = if (isNumber (_config >> "treatmentTime")) then {
     ["isNotInside", "isNotSwimming"]
 ] call EFUNC(common,progressBar);
 
+// add current action to display list
+private ["_medicalLevel", "_treatmentType", "_partName", "_itemClass", "_action", "_treatment"];
```

Comment:
use `private` keyword.

Reply:
updated code, but not sure this is what you meant. Plz check it out.